Sources/Purchasing/StoreKitAbstractions/SK2StoreTransaction.swift

```diff
@@ -16,14 +16,20 @@ import StoreKit
 @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
 internal struct SK2StoreTransaction: StoreTransactionType {
 
-    init(sk2Transaction: SK2Transaction, jwsRepresentation: String) {
+    /// - Parameter environmentOverride: Overrides the environment from the StoreKit 2 transaction.
+    /// Used to override the default `Xcode` environment when running tests.
+    init(sk2Transaction: SK2Transaction,
+         jwsRepresentation: String,
+         environmentOverride: StoreEnvironment? = nil
+    ) {
```

Comment:
Nit: for symmetry the closing `)` should be in the same line as the last parameter, or put the first parameter on its own line too:
```suggestion
    init(
         sk2Transaction: SK2Transaction,
         jwsRepresentation: String,
         environmentOverride: StoreEnvironment? = nil
    ) {
```